src/uct/ib/base/ib_md.c

```diff
@@ -185,6 +185,10 @@ static ucs_config_field_t uct_ib_md_config_table[] = {
      "Enable setting CPU affinity of memory registration threads.",
      ucs_offsetof(uct_ib_md_config_t, ext.mt_reg_bind), UCS_CONFIG_TYPE_BOOL},
 
+    {"MEM_RELAXED_ORDERING", "try",
+     "Allow system to reorder memory accesses to improve performance.\n",
+     ucs_offsetof(uct_ib_md_config_t, ext.mr_relaxed_order), UCS_CONFIG_TYPE_TERNARY},
```

Comment:
can you pls use UCS_CONFIG_TYPE_ON_OFF_AUTO
PCI_RELAXED_ORDER
"Enable relaxed ordering for PCI writes from device to memory, to improve performance on some systems."